habitat-lab/habitat/datasets/rearrange/modify_episodes_for_object_rearrange.py

```diff
@@ -633,8 +639,10 @@ def add_cat_fields_to_episodes(
         episode["candidate_goal_receps"] = get_candidate_receptacles(
             all_rec_goals, goal_rec_cat
         )
-        if enable_add_viewpoints and not add_viewpoints(
-            sim, episode, obj_idx_to_name, debug_viz
+        if (
```

Comment:
Currently this check evaluates 3 conditions. Lets say those conditions are represented by a, b and c. Right now you are doing:
(a and not b or c)
Can you add parenthesis to make the ordering of these evaluations clear? I think you are doing this:
((a and not b) or c)

Reply:
Cleaned this up now